README.md

```diff
@@ -19,3 +19,23 @@ with open(file_name, "r") as f:
     user_yaml = f.read()
     validate_user_yaml(user_yaml, file_name)
 ```
+
+## user.yaml conversion
+
+Convert a `user.yaml` file using the CLI:
```

Comment:
Maybe mention in some way that it's a conversion from the old format to the new format?